src/externals/mct/mct/m_GlobalSegMap.F90

```diff
@@ -1764,22 +1761,25 @@ subroutine active_pes_(GSMap, n_active, pe_list)
   character(len=*),parameter :: myname_=myname//'::active_pes_'
 
   integer :: count, i, n, ngseg, ierr
-  logical :: new
-  integer, dimension(:), allocatable :: temp_list
-  integer, dimension(:), allocatable :: perm
+  integer :: max_activepe, p
```

Comment:
I never touched it. The changes you refer to were made by Pat in commit e1ea1d1e4dfd6deba93346e18a6c67a2fd38a89c in the MCT repo.
Perhaps it came into CIME in an MCT update pull request.